meta-coral-bsp/conf/machine/coral-dev.conf

```diff
@@ -21,6 +21,8 @@ MACHINE_FEATURES += "wifi bluetooth optee qca6174"
 MACHINE_EXTRA_RDEPENDS += "\
     kernel-modules \
     libedgetpu \
+    firmware-qca6174 \
+    kernel-module-qca6174 \
```

Comment:
`kernel-module-qca6174` should not be needed here as we already include `kernel-modules` as a dependency

Reply:
Without the "kernel-module-qca6174" entry, the kernel module file qca6147.ko is not included in the rootfs.

Reply:
Ah, I see it now. It is an out-of-tree module. Makes sense